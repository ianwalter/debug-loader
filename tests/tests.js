const { test } = require('@ianwalter/bff')
const execa = require('execa')

test('debug-loader', async ({ expect }) => {
  const { stdout } = await execa('webpack')
  expect(stdout.split('\nHash:')[0]).toMatchSnapshot()
})
