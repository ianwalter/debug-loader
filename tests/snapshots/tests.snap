// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`debug-loader 1`] = `
"
🏁  [1mSTART[22m ./tests/fixtures/example.js 
    [1mexport default () => {[22m
[1m      console.log('Hello World!')[22m
[1m    }[22m
[1m[22m 
🏁  [1mEND[22m ./tests/fixtures/example.js
"
`;
